lesson_13/mihai_farcas/homework_polygons.py

```diff
@@ -0,0 +1,63 @@
+from math import sqrt
+
+
+class PerimetruMixin:
+
+    def __init__(self):
+        self.sides = None
+
+    def perimetru(self):
+        class_name = type(self).__name__
+        return 'The perimeter for {} is : {}'.format(class_name, sum(self.sides))
+
+
+class Polygons(PerimetruMixin):
```

Comment:
Placing the Mixin here will make the perimeter available for all shapes that inherit from Polygon.

Where we can move the mixin to have the perimeter method only on instances of Triangles?